internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,50 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	// Parse verbosity from CR
+	verbosityAnnotation := 0
+	if annot, exists := u.UnstructuredContent()["metadata"].(map[string]interface{})["annotations"]; exists {
+		if verbosityField, present := annot.(map[string]interface{})["ansible.sdk.operatorframework.io/verbosity"]; present {
+			var err error
+			verbosityAnnotation, err = strconv.Atoi(verbosityField.(string))
+			if err != nil {
+				logger.Error(err, "Unable to parse verbosity value from CR.")
+			}
+		}
+	}
+
+	// Parse verbosity from environment variable
+	verbosityEnvVar := 0
+	everb := os.Getenv("ANSIBLE_VERBOSITY")
+	if everb != "" {
+		var err error
+		verbosityEnvVar, err = strconv.Atoi(everb)
+		if err != nil {
+			logger.Error(err, "Unable to parse verbosity value from environment variable.")
+		}
+	}
+
 	// logger only the following for the 'Tasks' LogLevel
 	t, ok := e.EventData["task"]
 	if ok {
 		setFactAction := e.EventData["task_action"] == eventapi.TaskActionSetFact
 		debugAction := e.EventData["task_action"] == eventapi.TaskActionDebug
 
+		if verbosityAnnotation > 0 || verbosityEnvVar > 0 {
+			l.mux.Lock()
+			fmt.Println(e.StdOut)
```

Comment:
I think this should probably be preceded with a little log that makes it clear this is logging ansible stdout like here https://github.com/operator-framework/operator-sdk/pull/4961/files#diff-13da9e4e392cc2d59df5cd216407c953b17acf07bc614ed04d1b9d9385c6622aR104